codelists/tests/test_actions.py

```diff
@@ -176,6 +176,38 @@ def test_create_or_update_codelist_update_no_change_to_codes(
     assert codelist.methodology == "This is how we did it (updated)"
 
 
+def test_create_or_update_codelist_update_no_change_to_codes_with_force(
+    user, organisation, disorder_of_elbow_codes
+):
+    new_codelist_name = "test versions"
+    cl = actions.create_or_update_codelist(
+        owner=organisation,
+        name=new_codelist_name,
+        coding_system_id="snomedct",
+        codes=disorder_of_elbow_codes,
+        description="This is a test",
+        methodology="This is how we did it",
+        references=[{"text": "Some reference", "url": "http://example.com"}],
+        signoffs=[{"user": user.username, "date": "2021-04-21"}],
+    )
+    assert cl.versions.count() == 1
```

Comment:
Instead of asserting the number of versions here, you could use `with assert_difference(...)` as we do [here](https://github.com/opensafely-core/opencodelists/blob/0bbfc721c24f9e531022c4afe68342d5454b50ad/codelists/tests/test_actions.py#L144).